node/src/main/scala/coop/rchain/node/TerminalMode.scala

```diff
@@ -0,0 +1,16 @@
+package coop.rchain.node
+
+trait TerminalMode[F[_]] {
```

Comment:
Why do we need extra typeclass? I think this makes perfect sense to put it as part of `Console`

Reply:
I don't mind adding it to `Console` but it seems to be an orthogonal topic.